### DISCUSSION TITLE
Use Dreambooth and TI together for best person photorealistic likeness?

### DISCUSSION THREAD

**Discussion by Jonseed** [d8ahazard/sd_dreambooth_extension#455] [General]
I have found that the best way to achieve photorealism of a specific person is to use textual inversion and dreambooth together.

I train a TI on the dataset (with a unique token), and a model in dreambooth on the same dataset (with a different unique token), and then use them together in the prompt. Sometimes I have to adjust the weights of the tokens using [[]] or adjusting the cfg (usually down very low), in order to dial it in (even using inpainting to replace the face), but I have achieved some _amazing_ likeness using this approach. Neither token independently produces good likeness, but only when used together in this way.

Maybe it is my lack of good training in either TI or dreambooth, or maybe it is a new technique. I don't know. Has anyone else tried this?

**Comment by D4ndellion** [comment]
This was explored a bit in https://huggingface.co/blog/dreambooth

**Reply by Jonseed** [reply]
Yes! That is what I'm finding too, when used in combination the results can be better than either independently.

**Reply by Humanoidme** [reply]
Are you using the "train" tab for TI ?

**Reply by Jonseed** [reply]
> Are you using the "train" tab for TI ?

Yes

**Reply by Humanoidme** [reply]
Okay, will try. thanks

**Reply by Humanoidme** [reply]
How about first training using DB and then using the fine tuned model to train the TI and then using both for best outcome ?
